src/applications/disability-benefits/686/selectors/index.js

```diff
@@ -0,0 +1,9 @@
+// selectors for the `authorization686` state
+import { createStructuredSelector } from 'reselect';
+
+export const get686State = createStructuredSelector({
+  profileIsLoading: state => state.user.profile.loading,
+  isLoggedIn: state => state.user.login.currentlyLoggedIn,
+  isVerified: state => state.user.profile.verified,
```

Comment:
It looks like we might want to provide the user.profile.status as well to display appropriate error messages in the event that we cannot retrieve/find verified status, as done in RequireLoginView.

Reply:
We will also likely want to provide access to user.login.currentlyLoggedIn to determine where we route them in the error message (sign in or verify link)

Reply:
`user.login.currentlyLoggedIn` is already in there, but we'll also want the get access to the status of the disability rating check since we'll want to show a spinner if it's still getting the status.